version.json

```diff
@@ -6,6 +6,6 @@
     "Tracing": "1.1.0"
   },
   "Utilities": {
-    "Parameters": "0.0.1-preview"
+    "Parameters": "0.0.2-preview"
```

Comment:
Do we need to update the version? There are no code changes. If we push to nuget does it not update if the version is there already?

Reply:
I wouldn't say so

Reply:
There is a change in dll metadata at least

Reply:
Removed the version change, icon will be fixed with next release.